idl-parser/src/ffi/ast/visitor.rs

```diff
@@ -85,81 +85,131 @@ static VISITOR: Visitor = Visitor {
     visit_enum_variant,
 };
 
+macro_rules! defer {
```

Comment:
Should it be `deref` instead of `defer`? Or even `deref_data_ptr` so it matches to `deref_visitor`

Reply:
thanks, that was a typo